pkg/models/commandvalue_test.go

```diff
@@ -21,6 +21,153 @@ import (
 	"github.com/ugorji/go/codec"
 )
 
+// Test NewCommandValue function
+func TestNewCommandValue(t *testing.T) {
+	// Test Bool
+	cv, err := NewCommandValue("resource", 0, true, Bool)
+	test, err2 := NewBoolValue("resource", 0, true)
+	if err != nil || err2 != nil {
+		t.Errorf("Error creating command value")
+	}
+
+	if !reflect.DeepEqual(cv, test) {
+		fmt.Println(cv)
+		fmt.Println(test)
+		t.Errorf("CommandValue returned from NewCommandValue doesn't match NewBoolValue")
+	}
+
+	// Test int8
+	cv, err = NewCommandValue("resource", 0, int8(5), Int8)
+	test, err2 = NewInt8Value("resource", 0, int8(5))
```

Comment:
Should include testing the corresponding CommandValue get'ers result in the same value, i.e. `cv.Int8Value()` for this one.